server/src/controller/hello.controller.spec.ts

```diff
@@ -0,0 +1,36 @@
+import 'reflect-metadata';
```

Comment:
this import is a workaround until we come up with a more elegant solution to have annotated components tested